include/server/server.hpp

```diff
@@ -49,7 +49,12 @@ class Server
         boost::asio::ip::tcp::endpoint endpoint = *resolver.resolve(query);
 
         acceptor.open(endpoint.protocol());
+#ifdef SO_REUSEPORT
+        int one = 1;
+        setsockopt(acceptor.native_handle(), SOL_SOCKET, SO_REUSEADDR | SO_REUSEPORT, &one, sizeof(one));
```

Comment:
This relies on the particular Boost.ASIO header we're including to transitively include sys/types.h, sys/socket.h.